movement/movement.h

```diff
@@ -60,9 +60,10 @@ typedef union {
         // time-oriented complication like a sunrise/sunset timer, and a simple locale preference could tell an
         // altimeter to display feet or meters as easily as it tells a thermometer to display degrees in F or C.
         bool clock_mode_24h : 1;            // indicates whether clock should use 12 or 24 hour mode.
+        bool clock_24h_leading_zero : 1;    // indicates whether clock should leading zero to indicate 24 hour mode.
```

Comment:
A maintainer should clarify the project's policy on ABI stability before this can be reviewed. This will shift the meaning of all following bits which unfortunately breaks the binary interface for anyone saving and loading these settings to storage as a binary structure. There seems to be at least two open pull requests implementing this feature right now so I'd say there are people out there depending on such interfaces.

Reply:
Looks like movement 2.0 will store settings in LFS. Might even have settings for each watch face. This doesn't seem to be much of an issue anymore.